ftw/linkchecker/tests/helpers.py

```diff
@@ -47,6 +55,37 @@ def generate_test_data_excel_workbook():
     return xlsx_file
 
 
+def create_file_listing_block_with_report(portal, upload_two=False):
+    # build structure so that there is an excel report at
+    # linkchecker/reports which is the default in registry.xml
+    linkchecker_page = create(Builder('sl content page')
+                              .titled(u'linkchecker')
+                              .within(portal))
+    file_listing_block = create(Builder('sl listingblock')
+                                .titled(u'reports')
+                                .within(linkchecker_page))
+
+    # upload test asset to the file listing block
+    with open(TEST_REPORT, 'rb') as f_:
+        f_.seek(0)
+        data = f_.read()
+    file_ = api.content.create(
+        container=file_listing_block, type='File',
+        title=ASSET_NAME, file=data)
+    file_.setFilename(ASSET_NAME)
+    transaction.commit()
+
+    if upload_two:
```

Comment:
This is just a personal advice (you don't have to change it now): I tend to put refactoring (like moving this function to another module) in a separate commit, before adding new functionality. This makes diffing between the commits much easier, which is useful for debugging.

Reply:
I added a separate commit for moving the test function. Thanks for the advice. It makes totally sense to me.